projects/sandbox/data_generation/data_generation/utils.py

```diff
@@ -134,3 +134,54 @@ def inject_into_background(
     X += waveforms
 
     return X
+
```

Comment:
Why is this diff showing up as new lines? We already have this function: https://github.com/ML4GW/PE/blob/caa535e73bcf996f4aa7d2d4374c7a017c21b192/projects/sandbox/data_generation/data_generation/utils.py#L96-L136

Reply:
Oh OK. I see there is a `inject_into_background` and a `inject_into_random_background`. What is the difference between them?

Reply:
`inject_into_background` injects waveforms into a timeseries at specific times. This is used for generating injections for bilby.

`inject_into_random_background` randomly samples windows of `kernel_length` from background strain and injects signals into the center of those kernels. This is used for pp-plot injections

Reply:
They do completely different tasks. `inject_into_random_background` selects many random non-coincident kernels from a long stretch of background. `inject_into_background` injects deterministically into a long stretch of timeseries.